ServiceWiring.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+use MediaWiki\Logger\LoggerFactory;
+use MediaWiki\MediaWikiServices;
+
+return [
+	RedisCache::class => static function ( MediaWikiServices $services ) {
+		return new RedisCache(
+			$services->getMainConfig()->get( 'RedisServers' ),
```

Comment:
**issue** This will throw exception on undefined config value eg. for preview verify etc.